test/unit/plugin_test.rb

```diff
@@ -18,11 +18,11 @@
     before do
       # TODO: remove this, once the plugin loading is not based on ruby-load time
       # initialization
-      cli_reg.registry.clear
+      cli_reg.subcommands.clear
```

Comment:
btw: i don't think these lines go away with ruby-load time initialization (which we changed in https://github.com/chef/inspec/pull/434). e.g. if the second test (which adds subcommand) runs before the first one, the command is still in the registry.

imho: this line must stay in because right now we are dealing with a global singleton registry.